components/Common/Breadcrumbs/index.module.css

```diff
@@ -0,0 +1,36 @@
+.wrapper {
+  .list {
+    @apply flex
+    items-center
```

Comment:
tailwind tokens should be formatted with a space, see (Collaborator Guideline)[https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#example-of-a-css-module]

Reply:
[Collaborator Guideline](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#example-of-a-css-module):
"We require that you define one Tailwind Token per line, just as shown on the example above, since this improves readability"

Reply:
Yes, but if you look closely its also indented with a space like this:
```css
@apply flex
 items-center; 
```

Reply:
That's not what @umairraza96 said, Augustin. (Umair is correct here)

The point is that after the first `@apply` line each next line must have an extra level of indentation. i.e:

```css
@apply flex
  items-center
  ...
```

Reply:
Cool, is there a way to automate this process though? Not sure if prettier supports configuration that does just this. Normally I would expect that the formatter doing the heavy lifting so that we can just focus on the coding part.

I also recommend setting up a prettier sort import plugin, e.g, [prettier-plugin-sort-imports](https://github.com/trivago/prettier-plugin-sort-imports) to enforce consistent import style. :) @ovflowd

Reply:
We already merged a consistent import style, on ESLint side of things.

No there's no way to automate this without a Stylelint/Prettier plugin. I find it okay-enough to just ask for people to abide by it.